bin/internal/shared.bat

```diff
@@ -158,6 +159,10 @@ GOTO :after_subroutine
       EXIT 1
     )
     IF "%ERRORLEVEL%" NEQ "0" (
+      IF "%dart_sdk_retries%" EQU "3" (
```

Comment:
Can we make `"3"` a constant somewhere and use that in the error string and condition?

Reply:
Done